### PR TITLE
Generate product requirements from prompt

### PR DESCRIPTION
**Pull request by burieberry**
Changes in host and runtime-common:
- Update `createCard` action to allow passing in a format for the new card
- Add `runCommand` action: runs a skill on a given card

How to generate Product Requirements from a prompt:
1- Open AI App Generator card
2- Fill in the prompt fields and press button
3- A new PRD document will be created. Open the new room that is created in chat.
4- When bot has completed description, send a message to confirm changes.
5- Click Apply on the patch suggestion.

### CODE REVIEW

File: packages/host/app/components/operator-mode/interact-submode.gts [hunk 1]
```diff
@@ -320,6 +340,38 @@ export default class InteractSubmode extends Component<Signature> {
...
+        let newRoomId = await this.matrixService.createRoom(`New AI chat`, [
+          aiBotUsername,
+        ]);
+        await this.matrixService.sendMessage(
+          newRoomId,
+          message,
+          [card],
+          [commandCard],
+        );
+      } catch (e) {
```
Comment (IanCal) [inline packages/host/app/components/operator-mode/interact-submode.gts]: Do we need a try/catch if we just catch then rethrow the error?

**Review by IanCal — COMMENTED** [review]
Looks cool, going to test out locally.

Notes for the future - we'll be putting an additional layer in with CS-7116 to make commands more general so they're not just for sending messages

File: packages/host/app/components/operator-mode/interact-submode.gts (hunk 1 above)
Comment (burieberry) [reply packages/host/app/components/operator-mode/interact-submode.gts]: Nope, removed it. I used some of your ideas from your branch for cs-6995 for sending a message with the command. I see you're using command cards and tools there, feel free to update this as you see fit. Also added a disabled "Generate App Now" button on the PRD card if you want to add your code there.

Comment (burieberry) [reply packages/host/app/components/operator-mode/interact-submode.gts]: Ah and the skill card I have for product requirements is very basic, we can improve that too.

**Pull request merged**